gaffer-as-a-service/deploy/README-ui.md

```diff
@@ -0,0 +1,58 @@
+# Copyright 2021-2022 Crown Copyright
```

Comment:
isn't this the same as [this](https://github.com/gchq/gaffer-experimental/blob/a48b7eeda543a2ccfad32ddc9d22ce18f80e7d1f/gaffer-as-a-service/deploy/helm/charts/gaas-ui/README.md)?